.changeset/slow-lizards-sleep.md

```diff
@@ -0,0 +1,5 @@
+---
+'xstate': major
+---
+
+The `state._sessionid` property has been removed. It should be obtained directly from the actor: `actor.sessionId`.
```

Comment:
I'm not sure if we even need to expose this anyhow on the `actor` - but we can bikeshed this later on.